pkgs/os-specific/linux/kernel/linux-6.5.nix

```diff
@@ -3,7 +3,7 @@
 with lib;
 
 buildLinux (args // rec {
-  version = "6.5.1";
```

Comment:
You changed the `version`, but not the `hash`.
This broke `kernel_latest`, and  is already on nixos-unstable-small

Reply:
Ok, fixed by #254411

Reply:
> You changed the version, but not the hash.

Yeah, for some reason the update script broke and even though I look through the diff before pushing I missed it this time, sorry :facepalm: 

@K900 already fixed master, unless someone's faster, I'd cherry-pick their commit onto the backport PR tomorrow. Until then it stays a draft.